### ISSUE TITLE
PWA - Splash screen not shown on iPhone 12

### ISSUE THREAD

**Issue by hr015** [vaadin/flow#13617]
### Description of the bug

The splash screen is not shown in iPhone 12 devices. I have tried with multiple icon sizes, but any of them is recognised as splash screen

### Expected behavior

The installed application should show a splash screen after being launched. It works with iPhone SE and other Android devices.

### Minimal reproducible example

Create any PWA project (it can be the bakery starter) and include in the folder:
resources/META-INF.resources/icons

 all these icons:
 
icon-512x512.png
icon-1136x640.png
icon-640x1136.png
icon-2436x1125.png
icon-1125x2436.png
icon-1792x828.png
icon-828x1792.png
icon-1334x750.png
icon-750x1334.png
icon-2208x1242.png
icon-1242x2208.png
icon-1242x2688.png
icon-2688x1242.png
icon-2732x2048.png
icon-2048x2732.png
icon-2224x1668.png
icon-1668x2224.png
icon-2388x1668.png
icon-1668x2388.png
icon-2048x1536.png
icon-1536x2048.png

I also included the ones for the default icon:
icon-144x144.png
icon-192x192.png
icon-16x16.png
icon-32x32.png
icon-96x96.png
icon-128x128.png
icon-192x192.png
icon-256x256.png
icon-384x384.png
icon-512x512.png
icon-57x57.png
icon-60x60.png
icon-72x72.png
icon-76x76.png
icon-120x120.png
icon-128x128.png
icon-144x144.png
icon-152x152.png
icon-167x167.png
icon-180x180.png
icon-192x192.png
icon-256x256.png
icon-384x384.png
icon-512x512.png

### Versions

- Vaadin / Flow version: 14.8.3
- Java version: 11
- OS version: ios 15.14.1
- Browser version (if applicable):
- Application Server (if applicable):
- IDE (if applicable):

**Comment by TatuLund** [comment]
There is a related discussion here https://stackoverflow.com/questions/51329040/pwa-ios-splash-does-not-display-despite-following-instructions

**Comment by jouni** [comment]
My understanding is that these are the only icon/splash screen sizes Flow generates for the app: https://github.com/vaadin/flow/blob/master/flow-server/src/main/java/com/vaadin/flow/server/PwaRegistry.java#L488-L507

It clearly lacks many of the sizes required to support all iOS and iPadOS devices, including all landscape variations.

**Comment by taefi** [comment]
@jouni Thanks for your comment. 

Actually, I created a simple starter app which is available [here](https://vaadin-pwa-demo.herokuapp.com/), and tried it on an iPhone 11 Pro the resolution of which is 1125x2436 pixels. Currently, the platform generates the needed meta tag as follows:
```
<link sizes="1125x2436" rel="apple-touch-startup-image" media="(device-width: 375px) and (device-height: 812px) and (-webkit-device-pixel-ratio: 3)" href="icons/launch-1125x2436.png?-728253801" type="image/png">
```
Then I made a change for generating the a similar `<link...>` for iPhone 13 the resolution of which is 1170x2532 pixels and asked a friend to try it, and both had the same undesirable behaviour of showing a blank page for a sec or two, and then the application loads. 

I also changed the PwaRegistry.java to also add the `launch-1125x2436.png` in the icon array of [manifest.webmanifest](https://vaadin-pwa-demo.herokuapp.com/manifest.webmanifest) but still no luck.
